_plugins/generators/meta_generator.rb

```diff
@@ -0,0 +1,35 @@
+# require_relative '../../lib/utils/html_util'
```

Comment:
Can you delete this file if its no longer needed?

Reply:
@tcmacdonald Yes, mostly just wanted it in there for historical context, should I give the generator another go in the future.